vue_app/public/global.css

```diff
@@ -0,0 +1,95 @@
+@charset "UTF-8";

```

Comment:
recommend requiring from node modules if possible

Reply:
also you might be in to sanitize.css, defs worth a check-out, but has a bunch of accessibility problems attached if you use it. Still super cool project though!